Header header
float32 max_range
float32 fov
float32 range
uint8 status
float32 sigma
